statistics_calculations.py

```diff
@@ -117,3 +117,23 @@ def plot_percentage(counter):
     ax.legend(loc='center left', bbox_to_anchor=(1, 0.5))
     plt.xlim(0,100)
     plt.show()
+
+def plot_word(df, word, regex=True):
+    """ Counts the use of a specific word by month, and plot by user
+        Be careful, word might actually be a regex
+    """
+    by_month = df[df["text"].str.contains(r"{}($|\s)".format(word))]

```

Comment:
this will be really nitpicky, but calling it by month before you grouped by month is kind of a misnomer, no? Also I learned this word today (misnomer) and I feel like an English teacher using it, just sharing

Reply:
You're right. I've changed to word_df in all the function (I didn't want to open another df just for the grouping, so even after the grouping it's still word_df.

Also, did you look for a reason to use it? It is a great word I didn't know

Reply:
didn't look for it as much as it was in my head after reading it a day before and the perfect opportunity presented itself